test/1_stdlib/PrintForDebugger.swift

```diff
@@ -0,0 +1,58 @@
+// RUN: %target-run-simple-swift
```

Comment:
Please rename the testcase and the source file to be the same as the new enum name.